src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,27 @@
+package test.java;
+
+import main.java.Deadline;
+import main.java.Task;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+public class DeadlineTest {
+    @Test
+    public void deadline_markTest_success() {
+        Task t = new Deadline("test 1", "2019-10-15");
+        t.markDone();
+        assertEquals("X", t.getStatusIcon());
+    }
+
+    @Test
+    public void deadline_unmarkTest_success() {
+        Task t = new Deadline("test 1", "2019-10-15", true);
+        t.markUndone();
+        assertEquals(" ", t.getStatusIcon());
+    }
+
```

Comment:
Maybe you can try to test these methods more extensively, like marking first task, last task, task out of index etc.